cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/AuthBinding.java

```diff
@@ -116,8 +126,9 @@
    * @param role the role to which the actions need to granted
    * @param actions the actions which need to be granted
    * @throws RoleNotFoundException if the given role does not exist
+   * @throws Exception when there is any exception while running the client command to grant
    */
-  void grant(EntityId entityId, Role role, Set<Action> actions) throws RoleNotFoundException {
+  void grant(EntityId entityId, Role role, Set<Action> actions) throws RoleNotFoundException, Exception {
```

Comment:
you can simply have `throws Exception`, no need to have `RoleNotFoundException` in the declaration.